### ISSUE TITLE
Rename Options Classes

### ISSUE THREAD

**Issue by uruuru** [eclipse/elk#12]
We decided to rename the following classes specifying layout options:
```
LayoutOptions --> CoreOptions
layered.Properties --> LayeredOptions
```

I guess the same should be done for the ``force`` and ``mrtree`` property files.

**Comment by le-cds** [comment]
I think we should stay consistent and go with either Properties or Options.

**Comment by uruuru** [comment]
My bad, I corrected the naming.

**Comment by le-cds** [comment]
Sorry to bother you again, but how about we go with "Properties" since it's a collection of `IProperty` instances anyway?

**Comment by le-cds** [comment]
After a short internal discussion, I'm all for going with "Options". The rationale here is that we always talk about "layout options", but how they are represented is not all that interesting. They happen to be defined through `IProperty` objects, but that's not the point. Properties are a more generic mechanism that can be used to represent not only layout options, but other information as well.

TLDR: `CopreOptions` and `LayeredOptions` are perfectly fine choices.

**Comment by le-cds** [comment]
Note that we decided to do this only after we have finished migrating KIELER changes to ELK.

**Comment by uruuru** [comment]
Following your previous comment, should we also call it ``option`` (or ``layoutoption``) instead of ``property`` in the meta data language?

**Comment by le-cds** [comment]
Sounds sensible to me.
